core/templates/domain/suggestion/SuggestionThreadObjectFactorySpec.ts

```diff
@@ -55,6 +55,7 @@ describe('SuggestionThreadObjectFactory', () => {
       status: 'accepted',
       author_name: 'author',
       change: {
+        skill_id: '',
```

Comment:
It is better if we avoid empty string. can we use any suitable data?

Reply:
done